src/libraries/Microsoft.Extensions.Caching.Abstractions/src/Microsoft.Extensions.Caching.Abstractions.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="Microsoft.Extensions.Primitives" />
+    <ProjectReference Include="..\..\Microsoft.Extensions.Primitives\ref\Microsoft.Extensions.Primitives.csproj" />
```

Comment:
We started adding project references to the .sln filesin order to allow building from visual studio, you should add that for th e new project references

Reply:
I think we might not require that as we just build the latest netcoreapp in VS. Can you verify it.

Reply:
> I think we might not require that as we just build the latest netcoreapp in VS. Can you verify it.

With this work we want to build everything inside VS. I will update the slns, thanks.